### PR TITLE
Adjust column widths

### LINKED ISSUE #3831: Test Explorer Header widths not uniform
Header widths of Grouping by `Location` and `Outcome` don't use the same widths, they are independent.
Found while working on https://github.com/rubberduck-vba/Rubberduck/issues/3590#issuecomment-371970017

### PR DESCRIPTION
**Pull request by machthree**
I've added some code to the code behind - hopefully there isn't a simple way to link the column widths using XAML alone 

Closes #3831

### CODE REVIEW

File: Rubberduck.Core/UI/UnitTesting/TestExplorerControl.xaml.cs [hunk 1]
```diff
@@ -68,5 +69,31 @@ protected virtual void Dispose(bool disposing)
...
+                    // and then copy column widths from the old now hidden element on the subsequent invocation
+                    if (groupingGrid.IsVisible)
+                        _newGroupingGrid = groupingGrid;
+                    else
+                    {
+                        // copy column widths from previously visible grouping grid to ensure widths are consistent
+                        for (int columnIndex = 0; columnIndex < groupingGrid.Columns.Count; columnIndex++)
+                        {
+                            // copy column widths from current now hidden columns to the new visible columns
+                            _newGroupingGrid.Columns[columnIndex].Width = groupingGrid.Columns[columnIndex].Width;
```
Comment (MDoerner) [inline Rubberduck.Core/UI/UnitTesting/TestExplorerControl.xaml.cs]: This line throws an NRE on shutdown.

Comment (machthree) [reply Rubberduck.Core/UI/UnitTesting/TestExplorerControl.xaml.cs]: I've not seen an examples of this causing problems on shutdown although admittedly I've only been testing using Excel as a host and my tests have been rather simplistic. Can you suggest how to reproduce?

Comment (bclothier) [reply Rubberduck.Core/UI/UnitTesting/TestExplorerControl.xaml.cs]: I think to see this, you need to attach a debugger and try to shutdown Excel. The exception is likely being caught and handled (but in suboptimal manner).

Comment (machthree) [reply Rubberduck.Core/UI/UnitTesting/TestExplorerControl.xaml.cs]: Ignore that - just found how to recreate.

**Pull request merged**